apps/system/js/call_forwarding.js

```diff
@@ -179,18 +175,14 @@
      */
     _start: function() {
       this._slots = SIMSlotManager.getSlots();
-      this._defaultCallForwardingIconStates =
-        Array.prototype.map.call(this._slots, function() { return false; });
       this._callForwardingIconInitializedStates =
         Array.prototype.map.call(this._slots, function() { return false; });
 
-      this._callForwardingHelper =
-        SettingsHelper('ril.cf.enabled', this._defaultCallForwardingIconStates);
+      this._callForwardingHelper = SettingsHelper('ril.cf.enabled',
```

Comment:
`this._callForwardingIconInitializedStates` is used to record whether the icon has been initialized or not, so here we can simply do:
```
this._callForwardingHelper = SettingsHelper('ril.cf.enabled');
```